### PR TITLE
Tests: cleanup workdir before creating testsuite.

### PR DESCRIPTION
**Pull request by backstreetkiwi**
Small improvement to usage of beku.py:

* cleanup of the old testsuite
* print out the beku.py version

(see https://github.com/stackabletech/issues/issues/315 for details)

### CODE REVIEW

**Review by adwk67 — APPROVED** [review]
tested locally (also with parameters for `parallel`, `test`) that `tests/_work` is deleted. LGTM!

File: template/scripts/run_tests.sh [hunk 1]
```diff
@@ -10,6 +10,11 @@ if [ $beku_installed -ne 0 ]; then
   exit 1
 fi
 
+echo "Using beku version: $(pip list --disable-pip-version-check | grep beku)"
```
Comment (razvan) [inline template/scripts/run_tests.sh:13]:
```suggestion
echo "Using beku version: $(beku --version)"
```

**Review by razvan — APPROVED** [review]
lgtm

**Review by fhennig — APPROVED** [review]
LGTM

**Pull request merged**